torchbenchmark/util/model.py

```diff
@@ -188,12 +188,9 @@ def _get_batch_size_from_metadata(self) -> Optional[str]:
             return batch_size
 
     def _determine_batch_size(self, user_specified_batch_size=None):
-        # batch size priority for eval tests: not ALLOW_CUSTOMIZE_BSIZE > user specified > device specified > default
-        # batch size priority for train tests: not ALLOW_CUSTOMIZE_BSIZE > user specified > default
-
         self.batch_size = user_specified_batch_size
 
-        if not self.batch_size:
+        if not self.batch_size and getattr(self, "ALLOW_CUSTOMIZE_BSIZE", False):
```

Comment:
I thought this should be `getattr(self, "ALLOW_CUSTOMIZE_BSIZE", True)`: if `ALLOW_CUSTOMIZE_BSIZE` is not defined, by default we will believe the model batch size is customizable. In this case, we should use the per-device batch size by default.